src/components/checkbox-drawer/checkbox-drawer.js

```diff
@@ -0,0 +1,112 @@
+import '@brightspace-ui/core/components/colors/colors.js';
+import '@brightspace-ui/core/components/expand-collapse/expand-collapse-content.js';
+import '@brightspace-ui/core/components/inputs/input-checkbox.js';
+import '@brightspace-ui/core/components/inputs/input-checkbox-spacer.js';
+import '@brightspace-ui/core/components/icons/icon.js';
+import { css, html, LitElement } from 'lit';
+import { bodyCompactStyles } from '@brightspace-ui/core/components/typography/styles.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { LocalizeLabsElement } from '../localize-labs-element.js';
+import { SkeletonMixin } from '@brightspace-ui/core/components/skeleton/skeleton-mixin.js';
+
+class CheckboxDrawer extends LocalizeLabsElement(SkeletonMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			ariaLabel: { type: String, attribute: 'aria-label' },
```

Comment:
There seem to only be a few usages of this component -- would you be open to renaming this attribute while switching things over? We're not supposed to use the `aria-*` attributes on custom elements, which was a mistake we made in `<d2l-input-checkbox>` but is much harder to correct at this point.

Actually... combing through the search results, I don't think `aria-label` is used at all with this, so maybe it can just go away?

Reply:
Since this component is only ever used in my team's code in foundation-components ([here](https://github.com/BrightspaceHypermediaComponents/foundation-components/blob/main/features/discover/d2l-discover-rules.js#L202) and [here](https://github.com/BrightspaceHypermediaComponents/foundation-components/blob/main/features/discover/d2l-discover-approval-flows-config.js#L100)), I don't mind making this change along with any other breaking changes (like the one you mentioned in your other comment below). So if there are any other such changes that would help bring the component more in line with other labs or core component, just let me know and I can make those changes too 👍

Reply:
Amazing, thanks! Yeah I think I'd just remove this attribute and then update the event names to match the component name.